bridges/relays/client-substrate/src/transaction_tracker.rs

```diff
@@ -76,6 +78,21 @@ impl<C: Chain, E: Environment<C>> TransactionTracker<C, E> {
 		Self { environment, stall_timeout, transaction_hash, subscription }
 	}
 
+	// TODO (https://github.com/paritytech/parity-bridges-common/issues/2133): remove me after
+	// test client is implemented
```

Comment:
if we still need this, pls let's create a more descriptive issue for it. It's not clear what the preconditions for this TODO are.

Reply:
I summarized all TODOs here https://github.com/paritytech/parity-bridges-common/issues/3009 (I also pinged Slava)